docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md

```diff
@@ -0,0 +1,30 @@
+# Run jobs in case of insufficient capacity for region without Run shift policy flag
+
+Test verifies that run isn't restarted in another regions in case of insufficient capacity if initial region doesn't have Run shift policy flag.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
```

Comment:
Change to:
"Select `region1` (same as in step 4 of the preparations of case [3122_1](3122_1.md)) on the left panel"

Reply:
Done